exchange/non_bid_reason.go

```diff
@@ -22,3 +25,11 @@ func (n *NonBidReason) Val() NonBidReason {
 	}
 	return *n
 }
+
+func ErrorToNonBidReason(errorCode int) NonBidReason {
+	switch errorCode {
```

Comment:
Do you anticipate more cases to be added to this switch statement in the future? If not, can this be simplified to just check `if errorCode == errortypes.TimeoutErrorCode`?

Reply:
Yes, I could see we will need to incorporate more cases here in future based on the [Seat Non Bid community extension](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/seat-non-bid.md)